org.blender.BlenderLTS.json

```diff
@@ -0,0 +1,179 @@
+{
+    "id": "org.blender.BlenderLTS",
```

Comment:
What's the plan when new LTS is released, is this app going to track the latest LTS branch available?

Reply:
I assume I will build new releases as I see them :man_shrugging:

Reply:
Yeah, I don't mean new LTS releases of 2.93, but what when 3.4 or 3.5 is released and declared LTS.

Reply:
Same, I assume I'll just rebuild updates when that happens 🤷‍♂️